test/state/requests.hpp

```diff
@@ -34,14 +34,21 @@ struct Requests
         consolidation = 2,
     };
 
+    /// Raw encoded data of requests object: first byte is type, the rest is request objects.
+    evmc::bytes raw_data;
+
+    explicit Requests(Type _type, const evmc::bytes& data = {})
```

Comment:
Usage of `bytes` type seems inefficient. In the `system_call()` we are forced to create a temporary object of `bytes` just to be quicly replaced with "type + data". It may be better to use `bytes_view` here.

Reply:
true, I forgot that we can get rid of `bytes` on the `system_call()` side